modules/ROOT/pages/apikit-tutorial.adoc

```diff
@@ -4,16 +4,100 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: apikit, raml, tutorial, api, rest
 
-In this procedure, you build and deploy an APIkit project that imports a RAML. You run the API in Anypoint Studio, use API Console to simulate calls to the API, prepare the project for deployment, deploy the project to Anypoint Platform, and apply a policy to the API.
+In this workflow, you build an APIkit project that imports a RAML. You run the API in Anypoint Studio and use API Console to simulate calls to the API.
 
 Studio 6.0 or later is recommended for this procedure. The API you create covers two main tasks:
 
 * Updating vending machine data: to update the central database on each sale
 * Tracking inventory: to find out if a machine is low on stock
 
-. xref:apikit-create.adoc[Create a new project].
-. xref:apikit-run-test.adoc[Run and test the project].
-. xref:apikit-add-logic.adoc[Add logic to the project].
-. xref:apikit-deploy.adoc[Deploy the project].
-. xref:apikit-apply-policy.adoc[Apply a policy].
-. xref:apikit-test-policy.adoc[Test a Policy].
\ No newline at end of file
+== Create a New APIkit Project
+
+This procedure launches the new project and generates a Main flow with an HTTP Listener, an APIkit Router, and references to flows to handle errors. The project support REST resources and methods. These flows, aptly called skeleton flows, are the bare minimum needed to support the RAML specification. The RAML file imported by this procedure resides in `src/main/api`.
```

Comment:
Not sure why Main is capped in Main flow. Is it UI? If so, bold, if not, lowercase. 
The projects support**s**

These flows are called skeleton flows, as they are the bare minimum needed to support the RAML specification